markdown/reference/cli/admin_utilities/hawqregister.html.md.erb

```diff
@@ -200,6 +200,8 @@ group {
 <dt>-\\\-version  </dt> 
 <dd>Show the version of this utility, then exit.</dd>
 
+<dt>-l, -\\\-logdir \<logfile\_directory\>  </dt>
+<dd>Specifies the log directory that `hawq register` uses for logs. The default is `~/hawqAdminLogs/`.</dd>
```

Comment:
same comment as hawq extract ...  how about "Specifies the directory that `hawq register` uses for log files."?